iset.mm

```diff
@@ -39807,6 +39813,7 @@ The natural numbers (i.e. finite ordinals)
       ZATVNWMWOWSAWOWPWQVLWPWQHWOWNWQVJVIUEUFUGWNVJVLUKUHUIVMAIPWLWSAWKWPWHWRVI
       IBOVKIBOUJULUMWHAVPPQUNWCWJCVPIBUOVBVSVPUPVTWGWBWIVSVPDRWAWHAVSVPVSVPVKRU
       QURUSQVPWDUTVAVCVRIIMZVRKVQWTVRIVDVEIIBVFVGQ $.
+    $( $j usage 'peano3' avoids 'ax-setind'; $)
```

Comment:
peano3 -> peano5

Reply:
Thanks for noticing the copy paste error.

Fixed.